include/platform/mir/graphics/cursor.h

```diff
@@ -20,6 +20,7 @@
 
 #include "mir/geometry/size.h"
 #include "mir/geometry/point.h"
+#include "mir/graphics/renderable.h"
```

Comment:
I don't think we need this - we introduce a declaration on L32 that should be enough